crates/core_arch/src/wasm32/simd128.rs

```diff
@@ -1413,7 +1610,7 @@ pub unsafe fn i64x2_gt_s(a: v128, b: v128) -> v128 {
 #[inline]
 // #[cfg_attr(test, assert_instr(i64x2.le_s))] // FIXME llvm
 #[target_feature(enable = "simd128")]
-pub unsafe fn i64x2_le_s(a: v128, b: v128) -> v128 {
+pub unsafe fn i64x2_le(a: v128, b: v128) -> v128 {
     transmute(simd_le::<_, i64x2>(a.as_i64x2(), b.as_i64x2()))
 }
```

Comment:
`u64x2_{gt,lt,ge,le}` seem to be missing.

Reply:
Oh that's intentional, they're not actually [part of the spec](https://webassembly.github.io/simd/core/binary/instructions.html#simd-instructions)

Reply:
That seems like a glaring hole in the spec, is there a reason for it?

Reply:
I was not part of the discussions myself, but some digging reveals https://github.com/WebAssembly/simd/pull/414.

At the very least it's not accidental that they're left out!